tests/test_validation.py

```diff
@@ -0,0 +1,29 @@
+import copy
+
+import pytest
+
+from .configuration.config import valid_nhs_number1
+from .example_loader import load_example
+from .immunisation_api import ImmunisationApi
+
+
+@pytest.mark.nhsd_apim_authorization(
+    {
+        "access": "healthcare_worker",
+        "level": "aal3",
+        "login_form": {"username": "656005750104"},
+    }
+)
+@pytest.mark.debug
+def test_coarse_validation(nhsd_apim_proxy_url, nhsd_apim_auth_headers):
+    token = nhsd_apim_auth_headers["Authorization"]
+    imms_api = ImmunisationApi(nhsd_apim_proxy_url, token)
+
+    imms = copy.deepcopy(load_example("Immunization/POST-Immunization.json"))
```

Comment:
Why the copy? It looks like the load gives you your own object. Is there some caching maybe?

Reply:
It caused issues before when I needed to generate objects in a loop. I think it was caused by python caching the object. We don't call it multiple times here but I wanted to be safe.